packages/ssx-core/src/types.ts

```diff
@@ -10,9 +11,15 @@ import { ConnectionInfo } from 'ethers/lib/utils';
  */
 export interface SSXRouteConfig {
   /** Endpoint path. */
-  url: string;
+  url?: string;
   /** Endpoint request method. */
-  method: 'get' | 'post' | 'put' | 'delete';
+  method?: 'get' | 'post' | 'put' | 'delete';
+  /** Custom Operation.
+   * Replace the ssx function called with a function of your own
+   **/
+  customOperation?(
```

Comment:
The `customOperation` option was added as a way to override the default API operation of an endpoint. It's used as a core part of `ssx-react`'s NextAuth's flow

Reply:
Should we rename this to `customAPIOperation`?

Reply:
I agree with the rename, `HTTPMethod` sounds better for me